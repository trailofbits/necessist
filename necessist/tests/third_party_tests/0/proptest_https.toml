url = "https://github.com/nzeh/proptest"
rev = "7866f6d67afd9790ec6d7ee085db510cefe1a181"
subdir = "proptest"
test_files = ["src/sample.rs"]
full = true
