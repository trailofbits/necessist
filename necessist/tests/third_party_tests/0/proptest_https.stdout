30 candidates in 1 test file
$DIR/proptest/src/sample.rs: dry running
$DIR/proptest/src/sample.rs: mutilating
$DIR/proptest/src/sample.rs:428:30-428:52: `.new_tree(&mut runner)` nonbuildable
$DIR/proptest/src/sample.rs:428:61-428:71: `.current()` nonbuildable
$DIR/proptest/src/sample.rs:438:13-438:27: `sorted.sort();` passed
$DIR/proptest/src/sample.rs:438:19-438:26: `.sort()` nonbuildable
$DIR/proptest/src/sample.rs:441:13-441:43: `size_counts[value.len()] += 1;` failed
$DIR/proptest/src/sample.rs:441:30-441:36: `.len()` nonbuildable
$DIR/proptest/src/sample.rs:444:17-444:42: `value_counts[value] += 1;` failed
$DIR/proptest/src/sample.rs:457:44-457:56: `.enumerate()` nonbuildable
$DIR/proptest/src/sample.rs:475:22-475:44: `.new_tree(&mut runner)` nonbuildable
$DIR/proptest/src/sample.rs:475:53-475:63: `.current()` passed
$DIR/proptest/src/sample.rs:487:13-487:73: `counts[input.new_tree(&mut runner).unwrap().current()] += 1;` failed
$DIR/proptest/src/sample.rs:487:25-487:47: `.new_tree(&mut runner)` nonbuildable
$DIR/proptest/src/sample.rs:487:56-487:66: `.current()` nonbuildable
$DIR/proptest/src/sample.rs:490:42-490:54: `.enumerate()` nonbuildable
$DIR/proptest/src/sample.rs:536:33-536:55: `.new_tree(&mut runner)` nonbuildable
$DIR/proptest/src/sample.rs:537:13-537:52: `seen.insert(*tree.current().get(&col));` failed
$DIR/proptest/src/sample.rs:537:17-537:51: `.insert(*tree.current().get(&col))` nonbuildable
$DIR/proptest/src/sample.rs:537:30-537:40: `.current()` nonbuildable
$DIR/proptest/src/sample.rs:537:40-537:50: `.get(&col)` nonbuildable
$DIR/proptest/src/sample.rs:539:13-539:37: `while tree.simplify() {}` failed
$DIR/proptest/src/sample.rs:539:23-539:34: `.simplify()` nonbuildable
$DIR/proptest/src/sample.rs:552:38-552:50: `.into_iter()` nonbuildable
$DIR/proptest/src/sample.rs:552:50-552:60: `.collect()` nonbuildable
$DIR/proptest/src/sample.rs:556:33-556:55: `.new_tree(&mut runner)` nonbuildable
$DIR/proptest/src/sample.rs:557:13-557:55: `seen.insert(*tree.current().select(&col));` failed
$DIR/proptest/src/sample.rs:557:17-557:54: `.insert(*tree.current().select(&col))` nonbuildable
$DIR/proptest/src/sample.rs:557:30-557:40: `.current()` nonbuildable
$DIR/proptest/src/sample.rs:557:40-557:53: `.select(&col)` nonbuildable
$DIR/proptest/src/sample.rs:559:13-559:37: `while tree.simplify() {}` failed
$DIR/proptest/src/sample.rs:559:23-559:34: `.simplify()` nonbuildable
